imports/imports.go

```diff
@@ -70,6 +70,7 @@ import (
 	_ "dubbo.apache.org/dubbo-go/v3/protocol/dubbo3/health"
 	_ "dubbo.apache.org/dubbo-go/v3/protocol/dubbo3/reflection"
 	_ "dubbo.apache.org/dubbo-go/v3/protocol/grpc"
+	_ "dubbo.apache.org/dubbo-go/v3/protocol/grpc_new"
```

Comment:
The name should be determined.

Reply:
I used grpc_new to represent new protocol temporarily.